packages/storybook/stories/ChatComposite/CustomBehaviorExampleContainer.tsx

```diff
@@ -59,12 +59,7 @@ export const ContosoChatContainer = (props: ContainerProps): JSX.Element => {
   return (
     <>
       {adapter ? (
-        <ChatComposite
-          fluentTheme={props.fluentTheme}
-          locale={props.locale}
-          adapter={adapter}
-          visualElements={{ showErrorBar: true, showParticipantPane: true, showTopic: true }}
-        />
+        <ChatComposite fluentTheme={props.fluentTheme} locale={props.locale} adapter={adapter} />
```

Comment:
unrelated to the fix of this PR but npm run test under storybook is giving a warning about this (it should be failing but that's another issue)